app/Services/VersionService.php

```diff
@@ -42,36 +47,16 @@ public function getVersion(string $versionString) : Version
      *
      * @return Collection
      */
-    public function getAllVersions() : Collection
+    public function getAllVersions(): Collection
     {
         return Version::withTrashed()->get();
     }
 
-    /**
-     * Create a version based on the versions string
-     *
-     * @param string $versionString The version string to use
-     * @return bool True if the version is new, false otherwise
-     */
-    public function createOrUpdateVersion(string $versionString, bool $allowed) : bool
-    {
-        $version = Version::updateOrCreate(
-            [
-                'version' => $versionString,
-            ],
-            [
-                'deleted_at' => $allowed ? null : Carbon::now(),
-            ]
-        );
-
-        return $version->wasRecentlyCreated ? true : false;
-    }
-
     /**
      * Toggles whether or not a version is allowed to be used in production.
      *
-     * @throws VersionNotFoundException
      * @return void
+     * @throws VersionNotFoundException
```

Comment:
Docstring doesn't document all thrown exceptions.

Reply:
I think it only throws this one?

Reply:
ReleaseChannelNotFoundException is also thrown unless I'm going mad

Reply:
Don't think it does!